recipes/r-gdtools/meta.yaml

```diff
@@ -0,0 +1,89 @@
+{% set name = 'gdtools' %}
+{% set version = '0.1.4' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-{{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/{{ name }}_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/{{ name }}/{{ name }}_{{ version }}.tar.gz
+
+  md5: 62c2946391c7d1558e32025a6799cd0e
+
+build:
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: htmltools, testthat, fontquiver (>= 0.2.0)
+requirements:
+  build:
+    - r-base
+    - r-rcpp
+    - r-withr
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+    - xorg-libxrender
+    - xorg-libxext
+    - xorg-libxdmcp
+    - xorg-libxau
+    - cairo [osx]
+
+  run:
+    - r-base
+    - r-rcpp
+    - r-withr
+    - xorg-libxrender
+    - xorg-libxext
+    - xorg-libxdmcp
+    - xorg-libxau
+    - cairo [osx]
```

Comment:
Mh, but does the tool needs cairo? I suppose for drawing under some environments?